src/components/NFTPage/ItemCard.vue

```diff
@@ -155,6 +166,10 @@ export default {
       type: String,
       default: undefined,
     },
+    isWritingNFT: {
```

Comment:
So in HTML template can use `is-writing-nft` instead of `is-writing-n-f-t`
```suggestion
    isWritingNft: {
```

You may alias it with computed props 
```js
computed: {
  isWritingNFT() {
    return this.isWritingNft;
  },
},
```

Reply:
Looks better, thx!